### PR TITLE
Rename au -> australia for 2018

### PR DESCRIPTION
**Pull request by ericholscher**

### CODE REVIEW

**Comment by ericholscher** [comment]
@plaindocs renamed the `au` dir for 2018, and incorporated your PR.

File: docs/_data/config-australia-2018.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: Australia
 
-shortcode: au # prague australia?
+shortcode: australia # prague australia?
```
Comment (plaindocs) [inline docs/_data/config-australia-2018.yaml:3]: The only reason that we have shortcode and template code is that there are `na` and `eu` in a few places in the templates and datafiles. We should probably kill those once and for all and clean up at some point

**Review by plaindocs — APPROVED** [review]
Sounds good to me.

File: docs/_data/config-australia-2018.yaml (hunk 1 above)
Comment (ericholscher) [reply docs/_data/config-australia-2018.yaml:3]: Aye. The shortcode is used a lot of places, but it's basically just `name|lower`.

Comment (plaindocs) [reply docs/_data/config-australia-2018.yaml:3]: Right. It is now. But not when we had Portland, `na`. Which is why we have the hacky template code.

**Pull request merged**